src/topnav/topnav.component.tsx

```diff
@@ -22,6 +15,20 @@ export default function Topnav(props: TopnavProps) {
           alt="OpenMRS Logo"
         />
       </a>
+      <ul>
+        <li>
+          <NavLink to="/patient-search" activeClassName={styles.activeLink}>
+            <span>Patient Search</span>
+          </NavLink>
+        </li>
+      </ul>
+      <ul>
```

Comment:
We don't need to have to `<ul>` elements -- we can just have one `<ul>` with multiple `<li>` elements in it.

Reply:
Another bug related to this is that the `<nav>` that  is the parent of the two ul elements has display flex, justify-content space-between. See https://github.com/joeldenning/openmrs-esm-navbar/blob/master/src/topnav/topnav.css#L9.

When you have two children elements to a justify-content: space-between element, it pushes them to the far left and to the far right. But when you have three children elements, it spaces them out so the second element is centered. Which is not  what we want in these designs. See https://css-tricks.com/snippets/css/a-guide-to-flexbox/ for more info